### ISSUE TITLE
Improvements to results.plot_orbits

### ISSUE THREAD

**Issue by semaphoreP** [sblunt/orbitize#168]
Raised by a user:

- First, if I put rv_time_series = True, the colorbar does not show up. Even if I put show_colorbar = True
- Second, it always open a new window. Is there a way to specify which figure to put it in?
- Third, the label “epoch” is misleading. In the data table, the epoch is in MJD. Here, it is in year. I got quite confused because of that. Maybe the X axis should be renamed (Epoch [year]).

**Comment by sylacour** [comment]
2 additional remarks:
For 3), I would in fact plot the epoch as MJD. It makes everything more consistant.
Also, a minor bug, when I try to plot a corner with a fixed parameter, it tries to plot it, and crash. Maybe it should check which parameter are fixed before plotting (in the eventuality no param_list is given).

**Comment by sylacour** [comment]
And while we are talking about the corner plots issues, I have another one.

If I run 
$ results.plot_corner(results.labels[:10])
followed by
$ results.plot_corner(results.labels[9:])

The histogram of the first column of the first corner plot is the same as the histogram of the first column of the second corner plot (and we are missing a column). See attached plot:
[Corner_upto9.pdf](https://github.com/sblunt/orbitize/files/4582702/Corner_upto9.pdf)
[Corner_from9.pdf](https://github.com/sblunt/orbitize/files/4582703/Corner_from9.pdf)

**Comment by sblunt** [comment]
Thanks @sylacour. Do you need these changes urgently? If not, I might hold off making these changes for a few weeks. Let us know.

**Comment by sylacour** [comment]
There is no urgency. It is just feedback!
Currently I run the code pretty smoothly.

**Comment by sylacour** [comment]
I have run through the examples there until I got to:
"Controlling Concurrent Access to Resources”
Here, the software crash.
What is strange is that if I run the same code in a terminal, it works!
How do you run it yourself? Do you run it inside a PyCcharm terminal? Does it call “pydevconsole.py” on your computer too?

Here is the error message:

/Users/slacour/opt/anaconda3/bin/python /Applications/PyCharm.app/Contents/plugins/python/helpers/pydev/pydevconsole.py --mode=client --port=55628
import sys; print('Python %s on %s' % (sys.version, sys.platform))
sys.path.extend(['/Users/slacour/PycharmProjects/betapictoris'])
Python 3.7.4 (default, Aug 13 2019, 15:17:50) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.8.0 -- An enhanced Interactive Python. Type '?' for help.
PyDev console: using IPython 7.8.0
Python 3.7.4 (default, Aug 13 2019, 15:17:50) 
[Clang 4.0.1 (tags/RELEASE_401/final)] on darwin
In[2]: runfile('/Users/slacour/PycharmProjects/betapictoris/ttest3.py', wdir='/Users/slacour/PycharmProjects/betapictoris')
Now running: ['0', '1']
Now running: ['0', '1', '2']
Now running: ['0', '1', '2']
Now running: ['1', '2', '3']
Now running: <ListProxy object, typeid 'list' at 0x119443550; '__str__()' failed>
Process 4:
Process 2:
Process 5:
Process 6:
Process 7:
Process 8:
Process 9:
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
Traceback (most recent call last):
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/managers.py", line 811, in _callmethod
    conn = self._tls.connection
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/managers.py", line 811, in _callmethod
    conn = self._tls.connection
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/managers.py", line 811, in _callmethod
    conn = self._tls.connection
AttributeError: 'ForkAwareLocal' object has no attribute 'connection'
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/managers.py", line 811, in _callmethod
    conn = self._tls.connection
AttributeError: 'ForkAwareLocal' object has no attribute 'connection'
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/managers.py", line 811, in _callmethod
    conn = self._tls.connection
During handling of the above exception, another exception occurred:
AttributeError: 'ForkAwareLocal' object has no attribute 'connection'
Traceback (most recent call last):
During handling of the above exception, another exception occurred:
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
Traceback (most recent call last):
AttributeError: 'ForkAwareLocal' object has no attribute 'connection'
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
AttributeError: 'ForkAwareLocal' object has no attribute 'connection'
During handling of the above exception, another exception occurred:
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/slacour/PycharmProjects/betapictoris/ttest3.py", line 24, in worker
    pool.makeActive(name)
Traceback (most recent call last):
  File "/Users/slacour/PycharmProjects/betapictoris/ttest3.py", line 13, in makeActive
    self.active.append(name)
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "<string>", line 2, in append
Traceback (most recent call last):
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/managers.py", line 815, in _callmethod
    self._connect()
During handling of the above exception, another exception occurred:
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
Traceback (most recent call last):
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/managers.py", line 802, in _connect
    conn = self._Client(self._token.address, authkey=self._authkey)
  File "/Users/slacour/PycharmProjects/betapictoris/ttest3.py", line 27, in worker
    pool.makeInactive(name)
  File "/Users/slacour/PycharmProjects/betapictoris/ttest3.py", line 16, in makeInactive
    self.active.remove(name)
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/managers.py", line 811, in _callmethod
    conn = self._tls.connection
  File "<string>", line 2, in remove
  File "/Users/slacour/PycharmProjects/betapictoris/ttest3.py", line 24, in worker
    pool.makeActive(name)
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/managers.py", line 818, in _callmethod
    conn.send((self._id, methodname, args, kwds))
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/connection.py", line 206, in send
    self._send_bytes(_ForkingPickler.dumps(obj))
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/connection.py", line 492, in Client
    c = SocketClient(address)
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/connection.py", line 404, in _send_bytes
    self._send(header + buf)
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/connection.py", line 368, in _send
    n = write(self._handle, buf)
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/connection.py", line 619, in SocketClient
    s.connect(address)
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/Users/slacour/PycharmProjects/betapictoris/ttest3.py", line 13, in makeActive
    self.active.append(name)
  File "<string>", line 2, in append
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
ConnectionRefusedError: [Errno 61] Connection refused
AttributeError: 'ForkAwareLocal' object has no attribute 'connection'
BrokenPipeError: [Errno 32] Broken pipe
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/managers.py", line 815, in _callmethod
    self._connect()
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/slacour/PycharmProjects/betapictoris/ttest3.py", line 24, in worker
    pool.makeActive(name)
  File "/Users/slacour/PycharmProjects/betapictoris/ttest3.py", line 13, in makeActive
    self.active.append(name)
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/managers.py", line 802, in _connect
    conn = self._Client(self._token.address, authkey=self._authkey)
During handling of the above exception, another exception occurred:
  File "<string>", line 2, in append
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/connection.py", line 492, in Client
    c = SocketClient(address)
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/managers.py", line 815, in _callmethod
    self._connect()
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/connection.py", line 619, in SocketClient
    s.connect(address)
  File "/Users/slacour/PycharmProjects/betapictoris/ttest3.py", line 24, in worker
    pool.makeActive(name)
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/managers.py", line 802, in _connect
    conn = self._Client(self._token.address, authkey=self._authkey)
  File "/Users/slacour/PycharmProjects/betapictoris/ttest3.py", line 24, in worker
    pool.makeActive(name)
  File "/Users/slacour/PycharmProjects/betapictoris/ttest3.py", line 13, in makeActive
    self.active.append(name)
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/connection.py", line 492, in Client
    c = SocketClient(address)
Traceback (most recent call last):
ConnectionRefusedError: [Errno 61] Connection refused
  File "<string>", line 2, in append
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/connection.py", line 619, in SocketClient
    s.connect(address)
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/managers.py", line 815, in _callmethod
    self._connect()
  File "/Users/slacour/PycharmProjects/betapictoris/ttest3.py", line 13, in makeActive
    self.active.append(name)
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/managers.py", line 802, in _connect
    conn = self._Client(self._token.address, authkey=self._authkey)
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/connection.py", line 492, in Client
    c = SocketClient(address)
ConnectionRefusedError: [Errno 61] Connection refused
  File "<string>", line 2, in append
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/managers.py", line 815, in _callmethod
    self._connect()
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/connection.py", line 619, in SocketClient
    s.connect(address)
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/managers.py", line 802, in _connect
    conn = self._Client(self._token.address, authkey=self._authkey)
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/connection.py", line 492, in Client
    c = SocketClient(address)
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/connection.py", line 619, in SocketClient
    s.connect(address)
ConnectionRefusedError: [Errno 61] Connection refused
ConnectionRefusedError: [Errno 61] Connection refused
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/slacour/PycharmProjects/betapictoris/ttest3.py", line 24, in worker
    pool.makeActive(name)
  File "/Users/slacour/PycharmProjects/betapictoris/ttest3.py", line 13, in makeActive
    self.active.append(name)
  File "<string>", line 2, in append
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/managers.py", line 815, in _callmethod
    self._connect()
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/managers.py", line 802, in _connect
    conn = self._Client(self._token.address, authkey=self._authkey)
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/connection.py", line 492, in Client
    c = SocketClient(address)
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/connection.py", line 619, in SocketClient
    s.connect(address)
ConnectionRefusedError: [Errno 61] Connection refused
Process 1:
Traceback (most recent call last):
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/slacour/PycharmProjects/betapictoris/ttest3.py", line 27, in worker
    pool.makeInactive(name)
  File "/Users/slacour/PycharmProjects/betapictoris/ttest3.py", line 16, in makeInactive
    self.active.remove(name)
  File "<string>", line 2, in remove
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/managers.py", line 818, in _callmethod
    conn.send((self._id, methodname, args, kwds))
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/connection.py", line 206, in send
    self._send_bytes(_ForkingPickler.dumps(obj))
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/connection.py", line 404, in _send_bytes
    self._send(header + buf)
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/connection.py", line 368, in _send
    n = write(self._handle, buf)
BrokenPipeError: [Errno 32] Broken pipe
Now running: <ListProxy object, typeid 'list' at 0x119443550; '__str__()' failed>
Now running: <ListProxy object, typeid 'list' at 0x119443550; '__str__()' failed>
Process 3:
Traceback (most recent call last):
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/slacour/PycharmProjects/betapictoris/ttest3.py", line 27, in worker
    pool.makeInactive(name)
  File "/Users/slacour/PycharmProjects/betapictoris/ttest3.py", line 16, in makeInactive
    self.active.remove(name)
  File "<string>", line 2, in remove
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/managers.py", line 818, in _callmethod
    conn.send((self._id, methodname, args, kwds))
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/connection.py", line 206, in send
    self._send_bytes(_ForkingPickler.dumps(obj))
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/connection.py", line 404, in _send_bytes
    self._send(header + buf)
  File "/Users/slacour/opt/anaconda3/lib/python3.7/multiprocessing/connection.py", line 368, in _send
    n = write(self._handle, buf)
BrokenPipeError: [Errno 32] Broken pipe
Now running: <ListProxy object, typeid 'list' at 0x119443550; '__str__()' failed>
Now running: <ListProxy object, typeid 'list' at 0x119443550; '__str__()' failed>
Now running: <ListProxy object, typeid 'list' at 0x119443550; '__str__()' failed>
Now running: <ListProxy object, typeid 'list' at 0x119443550; '__str__()' failed>
Now running: <ListProxy object, typeid 'list' at 0x119443550; '__str__()' failed>
Now running: <ListProxy object, typeid 'list' at 0x119443550; '__str__()' failed>
Now running: <ListProxy object, typeid 'list' at 0x119443550; '__str__()' failed>

**Comment by sblunt** [comment]
The colorbar bug is addressed in PR #197.

**Comment by sblunt** [comment]
Hey @sylacour, I've addressed all the bugs and suggestions aside from this one:

> And while we are talking about the corner plots issues, I have another one.
> 
> If I run
> $ results.plot_corner(results.labels[:10])
> followed by
> $ results.plot_corner(results.labels[9:])
> 
> The histogram of the first column of the first corner plot is the same as the histogram of the first column of the second corner plot (and we are missing a column). See attached plot:
> [Corner_upto9.pdf](https://github.com/sblunt/orbitize/files/4582702/Corner_upto9.pdf)
> [Corner_from9.pdf](https://github.com/sblunt/orbitize/files/4582703/Corner_from9.pdf)

I'm a bit confused by this, since this looks like the intended behavior to me. As an example:

```
Python 3.8.5 
>> mylist = [1, 2, 3, 4, 5, 6, 7]
>> mylist[:2]
>> mylist[1:]
```

Let me know if I misunderstood. In particular, I'm going to close this issue (all other parts are addressed in PR #215), but feel free to reopen.
